Web/starter_project_barbies/components/home/SupportUs.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+function SupportUs() {
+  return (
+    <div className='mt-32 bg-gradient-to-r to-[#019CFA] from-[#264FAD]'>
```

Comment:
Its best to use the colors already defined in your tailwind.config file. You can add it there if its already.